gcloud_bigtable/google/bigtable/admin/cluster/v1/bigtable_cluster_service_api.py

```diff
@@ -0,0 +1,262 @@
+
+  # Page descriptors
+
+  # Service calls
+  def list_zones(self, name="", **kwargs):
+    """Lists the supported zones for the given project."""
+    list_zones_request = bigtable_cluster_service_messages_pb2.ListZonesRequest(
+        name=name, **kwargs)
+    return self.list_zones_callable()(list_zones_request)
+
+  def list_zones_callable(
+      self, is_retrying=None, max_attempts=None):
+    return api_callable.idempotent_callable(
+        self.stub.ListZones,
+        is_retrying=is_retrying,
+        max_attempts=max_attempts,
+        defaults=self.defaults)
+
+  def get_cluster(self, name="", **kwargs):
+    """Gets information about a particular cluster."""
+    get_cluster_request = bigtable_cluster_service_messages_pb2.GetClusterRequest(
+        name=name, **kwargs)
+    return self.get_cluster_callable()(get_cluster_request)
+
+  def get_cluster_callable(
+      self, is_retrying=None, max_attempts=None):
+    return api_callable.idempotent_callable(
+        self.stub.GetCluster,
+        is_retrying=is_retrying,
+        max_attempts=max_attempts,
+        defaults=self.defaults)
+
+  def list_clusters(self, name="", **kwargs):
+    """
+    Lists all clusters in the given project, along with any zones for which
+    cluster information could not be retrieved.
+    """
+    list_clusters_request = bigtable_cluster_service_messages_pb2.ListClustersRequest(
+        name=name, **kwargs)
+    return self.list_clusters_callable()(list_clusters_request)
+
+  def list_clusters_callable(
+      self, is_retrying=None, max_attempts=None):
+    return api_callable.idempotent_callable(
+        self.stub.ListClusters,
+        is_retrying=is_retrying,
+        max_attempts=max_attempts,
+        defaults=self.defaults)
+
+  def create_cluster(self, name="", cluster_id="", cluster=bigtable_cluster_data_pb2.Cluster(), **kwargs):
+    """
+    Creates a cluster and begins preparing it to begin serving. The returned
+    cluster embeds as its "current_operation" a long-running operation which
+    can be used to track the progress of turning up the new cluster.
+    Immediately upon completion of this request:
+     * The cluster will be readable via the API, with all requested attributes
+       but no allocated resources.
+    Until completion of the embedded operation:
+     * Cancelling the operation will render the cluster immediately unreadable
+       via the API.
+     * All other attempts to modify or delete the cluster will be rejected.
+    Upon completion of the embedded operation:
+     * Billing for all successfully-allocated resources will begin (some types
+       may have lower than the requested levels).
+     * New tables can be created in the cluster.
+     * The cluster's allocated resource levels will be readable via the API.
+    The embedded operation's "metadata" field type is
+    [CreateClusterMetadata][google.bigtable.admin.cluster.v1.CreateClusterMetadata] The embedded operation's "response" field type is
```

Comment:
:+1: It may be a pain unless you have something that can easily parse it from the comments in the `.proto` definition.

If you **can** parse it, you should reformat in Sphinx / RsT format to link within the generated classes.